akka-remote/src/main/java/akka/remote/artery/AeronErrorLog.java

```diff
@@ -74,8 +71,8 @@ public long logErrors(LoggingAdapter log, long sinceTimestamp)
                   log.error(String.format(
                       "Aeron error: %d observations from %s to %s for:%n %s",
                       observationCount,
-                      formatter.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(firstObservationTimestamp), timeZone)),
-                      formatter.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(lastObservationTimestamp), timeZone)),
+                      Helpers.timestamp(firstObservationTimestamp),
+                      Helpers.timestamp(lastObservationTimestamp),
```

Comment:
Always good to do it in a more consistent way.

`Helpers.timestamp` uses `MM/dd/yyyy HH:mm:ss.SSS` as a pattern. I rather like `yyyy-MM-dd` (easier sorting)... OTOH changing `StdOutLogger` behavior might break people's log analysis scripts so it's probably too late to change that now.